lab_5_scrapper/scrapper_config.json

```diff
@@ -1,9 +1,13 @@
 {
-    "seed_urls": [],
-    "headers": {},
-    "total_articles_to_find_and_parse": 0,
-    "encoding": "",
-    "timeout": 0,
+    "seed_urls": ["https://www.fontanka.ru/24hours_news.html"],
```

Comment:
one seed url is enough for 100 articles?

Reply:
unfortunately, there is only one such page. I have added thematic sections, but another alternative is an archive (https://www.fontanka.ru/arc/news.html)